src/App.js

```diff
@@ -1,25 +1,151 @@
-import logo from './logo.svg';
-import './App.css';
+import React, { useState } from "react";
+import Boxes from "./Components/Boxes";
+import "./App.css";
+import ColourPicker from "./Components/ColourPicker";
+
+const App = () => {
+  const [boxes, updateBoxes] = useState([
```

Comment:
If you're gonna have 3 rows of 3, doing what you did is not incorrect, however I think that you could definitely do this in a 2D array (An array of arrays) like this

```
const [boxes, updateBoxes] = useState([
  [
    { id: 1, highlight: false, colour: "yellow" },
    { id: 2, highlight: false, colour: "yellow" },
    { id: 3, highlight: false, colour: "yellow" }
  ],
  [
    { id: 4, highlight: false, colour: "yellow" },
    { id: 5, highlight: false, colour: "yellow" },
    { id: 6, highlight: false, colour: "yellow" }
  ],
  [
    { id: 7, highlight: false, colour: "yellow" },
    { id: 8, highlight: false, colour: "yellow" },
    { id: 9, highlight: false, colour: "yellow" }
  ],
])
```

But this will change the rest of the app